sdk/samples/iot/generate_certificate.cmd

```diff
@@ -21,30 +21,29 @@ type device_ec_cert.pem > device_cert_store.pem
 type device_ec_key.pem >> device_cert_store.pem
 
 echo.
+echo IMPORTANT:
 echo It is NOT recommended to use OpenSSL on Windows or OSX. Recommended TLS stacks are:
 echo Microsoft Windows SChannel: https://docs.microsoft.com/en-us/windows/win32/com/schannel
 echo OR
 echo Apple Secure Transport : https://developer.apple.com/documentation/security/secure_transport
 echo If using OpenSSL, it is recommended to use the OpenSSL Trusted CA store configured on your system.
 echo.
-echo If required (for example on Windows), download the Baltimore PEM CA from https://www.digicert.com/digicert-root-certificates.htm to the current folder.
```

Comment:
Why this is not needed anymore? This seems specific to Windows OS.

Reply:
Good question.  I took it out because the download instructions in this script are for ALL samples running on Windows, but cert samples are the only samples which will run this generate_certificate script.  I think it is most appropriate then to put this info in a common location, such as the README.md, where it can be referenced for all samples.  

I have included logging in the provisioning samples to alert the user if the env variable AZ_IOT_DEVICE_X509_TRUST_PEM_FILE is not set when it is needed.  (The other samples will be updated later to match.)

Reply:
That's a good idea to have a common doc for setting the certs. The only thing is how the information is getting scattered again. The feedback from customers in the past is that they need to take many hops to get all the info they need. I rather have to maintain more docs than to make the dev experience worse with not having all info in the same place.

Reply:
If the info you removed is applicable to just some samples, maybe instead of removing we should add a note explaining that. Makes it clearer, and the information is all there available at a single doc.